### ISSUE TITLE
Wires - Dragging a connection causes the screen to get blank, all shapes disappear

### ISSUE THREAD

**Issue by romartin** [ahome-it/lienzo-core#165]
Consider a wires connector with 2 connections, and so only 2 control points, and consider both connections linked to a shape. 
In this case, if dragging one of the connections over one of the other side,  causes all shapes to disappear, the canvas gets blank.

**Comment by danielzhe** [comment]
Hello! I tried to reproduce this one too to fix, but no success. :(

**Comment by romartin** [comment]
Hey @danielzhe ! no worries, yeah I'm almost sure we've already fixed this one, IIRC it was due to some NPE... so I'll proceed to closing it, we're always on time to reopen... :) 

FYI - I'll do an update to this repository code and do a review to the issues during next few days... as it's  a bit "out-dated" sorry. This is because we're working right now (just temporary!!) on a fork [here](https://github.com/kiegroup/lienzo-core/), so some new features and fixes are being included in our fork but not yet "back-ported" here. Will do during next few days, sorry for the inconveniences...

Thanks again for your collaboration!
